grid/social-media/followers/maps/twitter.suma

```diff
@@ -1,6 +1,6 @@
 // Twitter API Reference: https://developer.twitter.com/en/docs/twitter-api
 
-profile = "social-media/followers@1.1"
+profile = "social-media/followers@1.1-rev1"
```

Comment:
Do you actually need to do this? And will this work?

I thought you can publish a new map without any explicit versioning...

Reply:
it works, but versioning of maps is done automatically when publishing, this is not the correct way